Tests/HederaE2ETests/EthereumTransaction.swift

```diff
@@ -0,0 +1,113 @@
+        .data(using: .utf8)!
+
+    internal func testSignerNonce() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        let privateKey = PrivateKey.generateEcdsa()
+        let aliasAccountId = privateKey.toAccountId(shard: 0, realm: 0)
+
+        _ = try await TransferTransaction()
+            .hbarTransfer(testEnv.operator.accountId, Hbar(-1))
+            .hbarTransfer(aliasAccountId, Hbar(1))
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        _ = try await AccountInfoQuery().accountId(aliasAccountId).execute(testEnv.client)
+
+        let fileReceipt = try await FileCreateTransaction()
+            .keys([.single(testEnv.operator.privateKey.publicKey)])
+            .contents(testSmartContractBytecode)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let fileId = try XCTUnwrap(fileReceipt.fileId)
+
+        let contractReceipt = try await ContractCreateTransaction()
+            .adminKey(.single(testEnv.operator.privateKey.publicKey))
+            .gas(200000)
+            .constructorParameters(ContractFunctionParameters().addString("Hello from Hedera.").toBytes())
+            .bytecodeFileId(fileId)
+            .contractMemo("[e2e::ContractCreateTransaction]")
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let contractId = try XCTUnwrap(contractReceipt.contractId)
+
+        let chainId = Data.init(hex: "012a")
+        let nonce = Data.init(hex: "00")
+        let maxPriorityGas = Data.init(hex: "00")
+        let maxGas = Data.init(hex: "d1385c7bf0")
+        let gasLimit = Data.init(hex: "0249f0")
+        let to = Data.init(hex: try contractId.toSolidityAddress())
+        let value = Data.init(hex: "00")
+        let callData = ContractFunctionParameters().addString("new message").toBytes("setMessage")
+        let recoveryId = Data.init(hex: "01")
```

Comment:
```suggestion
        let chainId = Data(hex: "012a")
        let nonce = Data(hex: "00")
        let maxPriorityGas = Data(hex: "00")
        let maxGas = Data(hex: "d1385c7bf0")
        let gasLimit = Data(hex: "0249f0")
        let to = Data(hex: try contractId.toSolidityAddress())
        let value = Data(hex: "00")
        let callData = ContractFunctionParameters().addString("new message").toBytes("setMessage")
        let recoveryId = Data(hex: "01")
```
(`.init` is longhand for the syntax suggested)